src/Codeception/Module/AMQP.php

```diff
@@ -305,6 +305,76 @@ public function seeMessageInQueueContainsText($queue, $text)
         $this->assertContains($text, $msg->body);
     }
 
+    /**
+     * Count messages in queue.
+     *
+     * @param string $queue
+     *
+     * @return int
+     */
+    public function _countMessage($queue)
+    {
+        list($queue, $messageCount) = $this->getChannel()->queue_declare($queue, true);
+        return $messageCount;
+    }
+
+    /**
+     * Checks that queue have expected number of message
+     *
+     * ``` php
+     * <?php
+     * $I->pushToQueue('queue.emails', 'Hello, davert');
+     * $I->seeNumberOfMessageInQueue('queue.emails',1);
+     * ?>
+     * ```
+     *
+     * @param string $queue
+     * @param int $expected
+     */
+    public function seeNumberOfMessageInQueue($queue, $expected)
```

Comment:
I don't like being a grammar nazi, but this method should be named seeNumberOfMessage**s**InQueue.

Everything else is good.

Reply:
@Naktibalda  done